tests/test_0001_source_class.py

```diff
@@ -322,19 +319,14 @@ def test_xrootd(use_threads):
         assert one[:4] == b"root"
 
 
-@pytest.mark.skip(
-    reason="RECHECK: Run2012B_DoubleMuParked.root is super-flaky right now"
-)
 @pytest.mark.network
 @pytest.mark.xrootd
-@pytest.mark.parametrize("use_threads", [True, False], indirect=True)
-def test_xrootd_deadlock(use_threads):
+def test_xrootd_deadlock():
```

Comment:
Not really sure of what I did here, don't really understand this test. Please review @nsmith- in case I broke something important.

Reply:
I think this test is obsolete now that XRootD upstream has truly fixed the issues discussed in https://github.com/scikit-hep/uproot5/pull/59 and our CI is not testing older xrootd that needs the patch of https://github.com/scikit-hep/uproot5/blob/6b6fa9458b4ae46894e053dee25c56f678567f9d/src/uproot/extras.py#L105-L109
In principle we could in https://github.com/scikit-hep/uproot5/blob/6b6fa9458b4ae46894e053dee25c56f678567f9d/.github/workflows/build-test.yml#L62C11-L62C11 specify older xrootd to test but it seems a bit moot.

Reply:
Perhaps @chrisburr has input